src/core/app_config/index.ts

```diff
@@ -1,15 +1,32 @@
 import { i18n } from '~core/localization';
-import type { AppConfig } from './types';
+import { KONTUR_DEBUG } from '~utils/debug';
+import type {
+  AppConfig,
+  AppInfoResponse,
+  AppConfigEffective,
+  AppConfigGlobal,
+} from '~core/app/types';
 
 declare global {
   interface Window {
     konturAppConfig: AppConfig;
   }
 }
 
-export default (() => {
-  const konturAppConfig = globalThis.konturAppConfig ?? {};
-  return {
+const configs = {
+  global: {} as AppConfigGlobal,
+  custom: {} as AppInfoResponse,
+  merged: {} as AppConfigEffective,
+};
```

Comment:
in this options for configs it's not clear which one to choose, which one is the main one
can we say that global config is rather a default config? fallback?
"global" seems greater than "merged"